examples/samplers/simulated_annealing_sampler.py

```diff
@@ -50,20 +50,32 @@ def sample_relative(self, study, trial, search_space):
         # type: (InTrialStudy, FrozenTrial, Dict[str, BaseDistribution]) -> Dict[str, Any]
 
         if search_space == {}:
+            # The relative search space is empty (it means this is the first trial of a study).
             return {}
 
+        # The rest of this method is an implementation of Simulated Annealing (SA) algorithm.
         prev_trial = self._get_last_complete_trial(study)
+
+        # Update the current state of SA if the transition is accepted.
         if self._rng.uniform(0, 1) <= self._transition_probability(study, prev_trial):
             self._current_trial = prev_trial
 
+        # Pick a new neighbor (i.e., parameters).
         params = self._sample_neighbor_params(search_space)
+
+        # Decrease the temperature.
         self._temperature *= COOLDOWN_FACTOR
 
         return params
 
     def _sample_neighbor_params(self, search_space):
         # type: (Dict[str, BaseDistribution]) -> Dict[str, Any]
 
+        # Generate a sufficiently near neighbor (i.e., parameters).
+        #
+        # In this example, we treat the 10% region of the entire search space centered on
+        # the current point as the search space for sampling a sufficiently near neighbor.
```

Comment:
IMO, this sentence is complicated a little.
Maybe `define` makes the sentence a bit simpler.

> In this example, we define a sufficiently near neighbor as 10% region of the entire search space centered on the current point.